src/Marr.Data/DataHelper.cs

```diff
@@ -109,7 +109,7 @@ public static string GetMemberName<T>(this Expression<Func<T, object>> member)
             var memberExpression = (member.Body as MemberExpression);
             if (memberExpression == null)
             {
-                memberExpression = (member.Body as UnaryExpression).Operand as MemberExpression;
+                memberExpression = ((UnaryExpression) member.Body).Operand as MemberExpression;
```

Comment:
Not sure how many upstream changes have been made to Marr, but perhaps we should look at it and see if anything is of use? Changes for changes sake here seems unnecessary.